artifact/scripts/executor.hpp

```diff
@@ -0,0 +1,138 @@
+// Copyright 2023 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+#ifndef MENDER_ARTIFACT_V3_SCRIPT_EXECUTOR_HPP
+#define MENDER_ARTIFACT_V3_SCRIPT_EXECUTOR_HPP
```

Comment:
`v3` only shows up here. If this code is v3-specific, it should show up in many more places or it shouldn't be here (copy pasta?)

Reply:
Hmm, depends. For now the executor only runs `Artifact scripts` `v3`. But it's only a hard-coded value keeping it from doing otherwise.

I agree, I'll remove

Reply:
Seems, Kristian wants it in the `v3` folder, so moving this whole thing and keeping the def val